webapp/app/forms/steps/lotse/pauschbetrag.py

```diff
@@ -52,19 +56,130 @@ def calculate_pauschbetrag(has_pflegegrad=False, disability_degree=None, has_mer
         return 620
     elif disability_degree >= 20:
         return 384
-    
+
     return 0
 
 
-class StepPauschbetrag(LotseFormSteuerlotseStep):
-    
-    def get_overview_value_representation(self, value):
-        result = ''
-        
-        if value == 'yes':
-            result = str(self.get_pauschbetrag()) + ' ' + _('currency.euro')
-            
-        return result
+class DisabilityModel(BaseModel):
+    person_a_has_disability: Optional[str]
+    person_a_has_pflegegrad: Optional[str]
+    person_a_disability_degree: Optional[int]
+    person_a_has_merkzeichen_bl: Optional[bool]
+    person_a_has_merkzeichen_tbl: Optional[bool]
+    person_a_has_merkzeichen_g: Optional[bool]
+    person_a_has_merkzeichen_ag: Optional[bool]
+    person_a_has_merkzeichen_h: Optional[bool]
+
+    person_b_has_disability: Optional[str]
+    person_b_has_pflegegrad: Optional[str]
+    person_b_disability_degree: Optional[int]
+    person_b_has_merkzeichen_bl: Optional[bool]
+    person_b_has_merkzeichen_tbl: Optional[bool]
+    person_b_has_merkzeichen_g: Optional[bool]
+    person_b_has_merkzeichen_ag: Optional[bool]
+    person_b_has_merkzeichen_h: Optional[bool]
+
+    @root_validator()
+    def convert_disability_degree_to_int(cls, values):
+        values['person_a_disability_degree'] = int(values['person_a_disability_degree']) \
+            if values['person_a_disability_degree'] \
+            else None
+        values['person_b_disability_degree'] = int(values['person_b_disability_degree']) \
+            if values['person_b_disability_degree'] \
+            else None
+        return values
+
+
+class HasPauschbetragClaimPersonAPrecondition(DisabilityModel):
+    _step_to_redirect_to = StepMerkzeichenPersonA.name
+    _message_to_flash = _l('form.lotse.skip_reason.has_no_pauschbetrag_claim')
```

Comment:
Shouldn't this be a different message?
```suggestion
    _message_to_flash = _l('form.lotse.skip_reason.has_pauschbetrag_claim')
```

We probably need to add that one to messages.po, too